pages/index.vue

```diff
@@ -57,10 +57,11 @@
 export default {
   name: 'HomePage',
   asyncData({ app, error }) {
+    const lang = app.store.state.i18n.locale
```

Comment:
Could be `cons { locale } = app.store.state.i18n` 
And then in the `get` we could use `starts_with: `${locale}/events/`

Reply:
for this its the same